### PR TITLE
ASC-1623 upload to qtest

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: gating/periodic/upload_test_results_to_qtest.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "test_branch": "{{ RE_JOB_BRANCH }}",
+    "test_fork": "rcbops",
+    "test_commit": "{{ MOLECULE_GIT_COMMIT }}"
+  }
+}
+EOF
+
+## Functions -----------------------------------------------------------------
+
+source $(dirname ${0})/../../scripts/functions.sh
```
Comment (johnduarte) [inline gating/periodic/upload_test_results_to_qtest.sh]: There is no such directory for file in this repo.

**Review by johnduarte — CHANGES_REQUESTED** [review]

File: gating/periodic/upload_test_results_to_qtest.sh (hunk 1 above)
Comment (zreichert) [reply gating/periodic/upload_test_results_to_qtest.sh]: @johnduarte Updated

Comment (johnduarte) [reply gating/periodic/upload_test_results_to_qtest.sh]: @zreichert does the lack of that file prevent a needed environment variable from being set?  I was unable to understand the value of its inclusion.

**Review by johnduarte — APPROVED** [review]

**Review by ryan-rs — APPROVED** [review]

File: gating/periodic/upload_test_results_to_qtest.sh (hunk 1 above)
Comment (zreichert) [reply gating/periodic/upload_test_results_to_qtest.sh]: It was a bit of boilerplate that was copied from rpc-openstack.  I don't believe it to be necessary even for the equivalent script in rpc-openstack.

**Pull request merged**